<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addItems</class>
 <widget class="QDialog" name="addItems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_saveProduct">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Adding New Product&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>361</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>78</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="lineEdit_nameProduct"/>
    </item>
    <item row="0" column="5">
     <widget class="QComboBox" name="comboBox_typeProduct"/>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="lineEdit_productID"/>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="lineEdit_productWV"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Product Type :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Weight/Volume</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="comboBox_WV"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Product&lt;/p&gt;&lt;p&gt;Description :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="lineEdit_productPrice"/>
    </item>
    <item row="6" column="5">
     <widget class="QPlainTextEdit" name="plainTextEdit_DescriptionProd"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Product Name :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Price :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Product ID :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QSpinBox" name="quantity"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
